com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/app/discovery/ZigBeeNodeServiceDiscoverer.java

```diff
@@ -461,6 +457,9 @@ private boolean requestActiveEndpoints() throws InterruptedException, ExecutionE
      * @throws InterruptedException
      */
     private boolean requestNeighborTable() throws InterruptedException, ExecutionException {
+        if (!supportsManagementLqi) {
```

Comment:
Why is this necessary? If the task is removed due to the command not being supported, then it should not be necessary to check this again.

Reply:
Earlier safety check. I'll kill them